src/preload.ts

```diff
@@ -0,0 +1,18 @@
+import { contextBridge, ipcRenderer } from 'electron';
+
+const bridge: Bridge = {
+  send: (channel: string, data: any) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/ferrwan/brankas/pullRequest?prid=7906180)
```suggestion
  send: (channel: string, data: unknown) => {
```